src/App.test.js

```diff
@@ -209,26 +200,26 @@ describe('error message displaying at correct times', () => {
 //   })
 // }
 
-test('equal button is called one time', () => {
-  const x = 3;
-  const y = 4;
-  const onSubmit = jest.fn();
-  render(<App onSubmit={"="} />);
-  const { getByTestId } = render(<App />);
+// test('equal button is called one time', () => {
```

Comment:
Why not delete this code rather than commenting it out?

Reply:
Yes! Fixed.